scripts/strings/french/sys_notify.lua

```diff
@@ -0,0 +1,67 @@
+-- OPENTOMB SYSTEM CONSOLE NOTIFICATIONS
```

Comment:
Sys notyfy not uses LUA strings: it is hardcoded... LUA translation ONLY for game and UI... system is only build in (English is hardcoded for sysstrings: reason: if something goes wrong in LUA then sys warning strings will be loosed + it is little bit slower)